### DISCUSSION TITLE
Errors encountered in Autobuild

### DISCUSSION THREAD

**Discussion by geteff1** [microsoft/autogen#1359] [Q&A]
Encountered errors with Autobuild, plz help. Thanks!
`from autogen.agentchat.contrib.agent_builder import AgentBuilder

builder = AgentBuilder(config_file_or_env=config_file_or_env, builder_model='gpt-4-1106-preview', agent_model='gpt-4-1106-preview')`
AttributeError: type object 'AssistantAgent' has no attribute 'DEFAULT_DESCRIPTION'

**Comment by ekzhu** [comment]
Please post your python version and pyautogen version. 

@JieyuZ2 @LinxinS97

**Reply by geteff1** [reply]
Never mind, the issue has been resolved. Thanks anyway.
